_posts/2020-10-09-torch-optim/torch_network_with_optim.Rmd

```diff
@@ -0,0 +1,269 @@
+---
+title: "Optimizers in torch"
+description: > 
+ Today, we wrap up our mini-series on torch basics, adding to our toolset two abstractions we hadn't made use of before: loss functions and optimizers.
```

Comment:
```suggestion
 Today, we wrap up our mini-series on torch basics, adding to our toolset two abstractions: loss functions and optimizers.
```

Reply:
That they weren't used before is implied by the fact that you're adding them to the toolset 👍

Reply:
lol right